### DISCUSSION TITLE
Removing unwanted & unused features from Opencart

### DISCUSSION THREAD

**Discussion by kanenas** [opencart/opencart#14120] [Q&A] [answered]
If in Opencart (for example v3.0.4.0), I remove the following features:
- Affiliates
- Recurring payments
- Downloadable product
- Marketing + mail
- Returns
- Most shipping methods
- Most payments methods

Will it make it faster?

**Comment by mhcwebdesign** [comment, marked as answer]
Probably not. If you want to discuss it further please use the OpenCart forum.
